toolkit/resources/manifests/package/ccache-configuration.json

```diff
@@ -0,0 +1,74 @@
+{
+  "remoteStore": {
+    "type": "azure-blob-storage",
+    "tenantId": "",
+    "userName": "",
+    "password": "",
```

Comment:
Is the idea that a local user doesn't need a password to download, and a pipeline will already be authenticated and won't need to login again?

Reply:
Correct re. a local user won't need a password to download. However, it is because download is allowed to be anonymous. So, a dev building on a local box will automatically get the benefit of ccache remote archive without haivng to log-in.